### PR TITLE
feat: Removes deleted content items

### PR DESCRIPTION
**Pull request by tomthornton**
Once content items are deleted in rock, their corresponding row in Postgres is deleted along with any media attached to it.

This should also delete related `ContentItemConnections` and `ContentTags`

### CODE REVIEW

**Comment by tomthornton** [comment]
@vinnyjth one thing that this PR doesn't handle is accounting for child cover images. We can either include that in this PR or a separate one, but I wanted to discuss how we should handle it.

**Review by vinnyjth — CHANGES_REQUESTED** [review]
Your code looks fantastic. We'll have to solve for one more case before we can merge this though. 
```
[2021-07-19 18:22:47,465] {dbapi.py:180} INFO - Running statement: 
            DELETE FROM "contentItems"
            WHERE id = ANY(%s::uuid[])
        , parameters: (['d1e0be33-f83b-4ca9-b5f3-7849a52f1a30'],)
[2021-07-19 18:22:47,516] {taskinstance.py:1454} ERROR - update or delete on table "contentItems" violates foreign key constraint "contentItems_parentId_fkey" on table "contentItems"
DETAIL:  Key (id)=(d1e0be33-f83b-4ca9-b5f3-7849a52f1a30) is still referenced from table "contentItems".
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1112, in _run_raw_task
    self._prepare_and_execute_task_with_callbacks(context, task)
  File "/usr/local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1284, in _prepare_and_execute_task_with_callbacks
    result = self._execute_task(context, task_copy)
  File "/usr/local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1314, in _execute_task
    result = task_copy.execute(context=context)
  File "/usr/local/lib/python3.7/site-packages/airflow/operators/python.py", line 117, in execute
    return_value = self.execute_callable()
  File "/usr/local/lib/python3.7/site-packages/airflow/operators/python.py", line 128, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/usr/local/airflow/dags/rock_deleted_content_items_dag.py", line 47, in remove_deleted_content_items
    """, True, (deletedContentItems,))
  File "/usr/local/lib/python3.7/site-packages/airflow/hooks/dbapi.py", line 182, in run
    cur.execute(sql_statement, parameters)
psycopg2.errors.ForeignKeyViolation: update or delete on table "contentItems" violates foreign key constraint "contentItems_parentId_fkey" on table "contentItems"
DETAIL:  Key (id)=(d1e0be33-f83b-4ca9-b5f3-7849a52f1a30) is still referenced from table "contentItems".
```

The steps I took to test were. 

1. Created a new content item in the campaign channel (https://rock.apollos.app/page/343?ContentChannelId=9)
2. Gave that item a cover image and a persona. Added a child item.
3. Ran the backfill operator to pick that item up in Postgres.
4. Deleted the item in rock (after confirming it had shown up in postgres) 
5. Ran the backfill operator to delete the item

I think we'll need to go through and wipe out the `parentId` of all items that have a parent set to the deleted item. Also open to other options :) Not sure if you accomplish what I mentioned above through a foreignKey constraint.

File: dags/rock_deleted_content_items_dag.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        keepalives_idle=30,
+        keepalives_interval=10,
+        keepalives_count=5
+    )
+
+    postgresContentItems = pg_hook.get_records('SELECT id, "originId" FROM "contentItems"')
+
+    params = {
+        "$select": "Id",
+        "loadAttributes": "expanded",
```
Comment (vinnyjth) [inline dags/rock_deleted_content_items_dag.py:25]: Not a big deal, but `loadAttributes` doesn't do anything here :)

**Comment by tomthornton** [comment]
@vinnyjth this pr should help [#2068](https://github.com/ApollosProject/apollos-apps/pull/2068) :)

**Comment by vinnyjth** [comment]
Need https://github.com/ApollosProject/apollos-apps/pull/2066 merged before I can finish testing.

**Comment by vinnyjth** [comment]
Was able to test, and it worked great :D nice work!

**Review by vinnyjth — APPROVED** [review]

**Pull request merged**